src/index.html

```diff
@@ -7,12 +7,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <section class="product-card__product-image"></section>
+
+      <p class="product-card__product-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="product-card__product-code">Product code: 195434</p>
+
+      <section class="product-card__product-reviews">
+        <section class="product-stars">
+          <ul class="product-stars__stars-line">
+            <li class="product-stars__star"></li>
+            <li class="product-stars__star"></li>
+            <li class="product-stars__star"></li>
+            <li class="product-stars__star"></li>
+            <li class="product-stars__star"></li>
```

Comment:
The `product-stars` block should be defined as a separate BEM block. Ensure that the BEM naming convention is consistently applied across your HTML and CSS files.